src/domain/wallets/index.types.d.ts

```diff
@@ -166,11 +166,11 @@ type DepositFeeCalculator = {
 }
 
 type OnChainWithdrawalFeeArgs = {
-  onChainFee: Satoshis
-  walletFee: Satoshis
+  minerFee: Satoshis
+  bankFee: Satoshis
```

Comment:
Can be `dealerFee`... or are we also introducing a new `Bank` concept?

Reply:
It was the bankFee elsewhere already. it's not a new concept. it's just the fee the bankowner charges for withdrawal on top of the miner fees.

Reply:
I think we could introduce the notion of `"Internal"` and `"external"` fees. the first one would be for fee the bankowner charges, the seconds are fees paid to the network (miner fees or routing fees for instance). 

it would make it more clear how much fees we are charging.

because for lightning for instance now we have inboundLiquidityFees (not yet implemented). those fees will be internal, and I think it would be more clear if it's named `InternalInboundLiquidityFees`, otherwise it may not be obvious to someone looking at the code that is a fee charged by the bankowner only 

(there are actually no fee for inbound on lightning, but this may not be obvious to everybody)

for sending on lightning, then there would be something like "InternalBankFee" and "ExternalRoutingFee"

thoughts @vindard @dolcalmi @bodymindarts  ?

Reply:
I like the distinction of `Internal`/`External` fees as a naming convention.